src/GUI_draft/templates/create_project.html

```diff
@@ -5,26 +5,48 @@
 <!-- WRITE HTML CODE WITHIN THESE block content TAGS -->
 {% block content %}
 
+<!-- Currently dummy data, but in the actual app this data comes from a database -->
+{% set project_parameters={'Project Name': 'Berlin Energy Cell', 'Project Id': '34drg3', 'Country': 'Germany', 'Latitude': '50.99999', 'Longitude': '420.44747'} %}
```

Comment:
Each parameters should come with values for the properties from the "input_parameters.csv" file, i.e., default, type, unit, description etc ...